pkg/runners/builtin_runner/bitwise.go

```diff
@@ -0,0 +1,106 @@
+package builtinrunner
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+type BitwiseInstanceDef struct {
+	Ratio      *uint
+	TotalNBits uint
+}
+
+type BitwiseBuiltinRunner struct {
+	base             memory.Relocatable
+	included         bool
+	CellsPerInstance uint
+	Ratio            *uint
+	BitwiseBuiltin   BitwiseInstanceDef
+}
+
+func NewBitwiseBuiltinRunner(instance_def BitwiseInstanceDef, included bool) BitwiseBuiltinRunner {
+	return BitwiseBuiltinRunner{
+		base:             memory.NewRelocatable(0, 0),
+		included:         included,
+		CellsPerInstance: 5,
+		Ratio:            instance_def.Ratio,
+		BitwiseBuiltin:   instance_def,
+	}
+}
+
+func (b *BitwiseBuiltinRunner) Base() memory.Relocatable {
+	return b.base
+}
+
+func (b *BitwiseBuiltinRunner) Name() string {
+	return "bitwise_builtin"
+}
+
+func (b *BitwiseBuiltinRunner) InitializeSegments(segments *memory.MemorySegmentManager) {
+	b.base = segments.AddSegment()
+}
+
+func (b *BitwiseBuiltinRunner) InitialStack() []memory.MaybeRelocatable {
+	if b.included {
+		return []memory.MaybeRelocatable{*memory.NewMaybeRelocatableRelocatable(b.base)}
+	} else {
+		return []memory.MaybeRelocatable{}
+	}
+}
+
+func (b *BitwiseBuiltinRunner) DeduceMemoryCell(address memory.Relocatable, segments *memory.Memory) (*memory.MaybeRelocatable, error) {
+	index := address.Offset % b.CellsPerInstance
+	if index < 1 {
```

Comment:
This should be <= 1. 
Even better, we could set the constant `INPUT_CELLS_PER_INSTANCE = 2` and use it in this if

Reply:
INPUT_CELLS_PER_INSTANCE shouldn't be 1 ?

Reply:
or you mean setting in two, because it contemplates the case <= 1 ?

Reply:
Yes, the inumber nput cells for bitwise is 2 so adding the constant and using it here will make it easier to document